rabbitmq/init.sls

```diff
@@ -1,6 +1,6 @@
 include:
-  - rabbitmq.install
-  - rabbitmq.plugins
-  - rabbitmq.os_tweaks
-  - rabbitmq.configure
-  - rabbitmq.permissions
```

Comment:
Is there a reason for removing `.permissions` from the init.sls?

Reply:
I actually didn't remove it, just made it a relative path.

Reply:
Roger that. I overlooked that in the diff :)